File: is_array/is_array.java
public class is_array {
    // ATOMIC_FUNCTION:  isArray()
    // TAGS: string,list
    // LANG:  java
    // FUNCTION: isArray()



    public static void main(String args[]) {
        String[] start_val = {"hello","world"};
        Boolean expected_end_val=true;

        Boolean end_val = start_val.getClass().isArray();

        // TEST
        if(end_val.equals(expected_end_val)) {
            System.out.println("PASS");
        } else {

            System.out.println("FAIL");
        }
    }
}

File: is_array/is_array.py

# ATOMIC_FUNCTION: is_array(value)
# TAGS: list
# LANG:  python
# FUNCTION: is_array(value)
def is_array(value):
	return isinstance(value, list)

start_val = ["hello","world"]
expected_end_val = True

end_val = is_array(start_val)

# TEST
if(end_val == expected_end_val):
    print("PASS")
else:
    print("FAIL")







